pkg/controller/cabundleinjector/starter.go

```diff
@@ -91,3 +96,13 @@ func StartCABundleInjector(ctx context.Context, controllerContext *controllercmd
 
 	return nil
 }
+
+func hasSupportedInjectionAnnotation(obj v1.Object, supportedAnnotations []string) bool {
+	annotations := obj.GetAnnotations()
+	for _, key := range supportedAnnotations {
+		if annotations[key] == "true" {
```

Comment:
`HasInjectCABundleAnnotation` used `strings.EqualFold`; is it all right to replace that with `==`?

Reply:
Good catch! Fixed by #103